### PR TITLE
Create Bundler March Updates

### PR DESCRIPTION
**Pull request by lauragift21**
Blog post with March 2020 Recap

Thanks so much for the contribution!
To make reviewing this PR a bit easier, please fill out answers to the following questions.

### What was the end-user problem that led to this PR?

The problem was...

### What was your diagnosis of the problem?

My diagnosis was...

### What is your fix for the problem, implemented in this PR?

My fix...

### Why did you choose this fix out of the possible options?

I chose this fix because...

### CODE REVIEW

File: source/blog/2020-04-27-march-monthly-update.html.markdown [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "March Bundler Update"
+date: 2020-04-27
+tags:
+author: Gift Egwuenu
+---
+
+Welcome to the Bundler monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to Bundler in March.
```
Comment (olleolleolle) [inline source/blog/2020-04-27-march-monthly-update.html.markdown:8]:
```suggestion
Welcome to the Bundler monthly update! As part of our efforts at [Ruby Together](https://rubytogether.org), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to Bundler in March.
```

File: source/blog/2020-04-27-march-monthly-update.html.markdown [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Not just the repositories have changed, either: the Bundler and RubyGems teams have merged as well. It’ll take us some time to work through 20 years worth of documentation and webpages, but we’ll be combining and updating them as fast as we can.
+
+From this point forward, Bundler news will be reported together with RubyGems news. Hooray! 🎉
+
+---
+Interested in contributing to Bundler? We always welcome contributions in the forms of triaging bugs, adding new features, writing docs, and engaging with the wider community. Visit the [Bundler Contributor Guidelines](https://github.com/bundler/bundler/blob/master/doc/contributing/README.md) on GitHub to get started. Don’t have time to contribute, but want to support our work? Sign up as a member of [Ruby Together](https://rubytogether.org/) to help fund our work to keep Bundler working for everyone.
```
Comment (olleolleolle) [inline source/blog/2020-04-27-march-monthly-update.html.markdown:17]:
```suggestion
Interested in contributing to Bundler? We always welcome contributions in the forms of triaging bugs, adding new features, writing docs, and engaging with the wider community. Visit the [Bundler Contributor Guidelines](https://github.com/rubygems/rubygems/tree/master/bundler/doc/contributing) on GitHub to get started. Don’t have time to contribute, but want to support our work? Sign up as a member of [Ruby Together](https://rubytogether.org/) to help fund our work to keep Bundler working for everyone.
```

**Comment by olleolleolle** [comment]
I marked two links that could be updated to match the current state.

**Comment by lauragift21** [comment]
Thanks for pointing that out @olleolleolle. Will push a fix.

**Comment by indirect** [comment]
Looks great, thank you!

**Pull request merged**